keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -225,7 +207,7 @@ The last byte represents the encoding style, with 0 meaning that the rest of the
 type EncryptedObject struct {
     TypeMeta `json:",inline" protobuf:"bytes,1,opt,name=typeMeta"`
 
-    EncryptResponse EncryptResponse `protobuf:"bytes,2,opt,name=encryptResponse"`
+    KeyId string `protobuf:"bytes,2,opt,name=keyId"`
```

Comment:
Instead of just keyId, should we just store the entire annotations map here?

Reply:
If we store the entire annotation here, how will it be used later? Anything extra we add here can reduce available etcd object size for the actual data right?